lib/core/src/lib/notifications/components/notification-history.component.spec.ts

```diff
@@ -32,14 +33,17 @@ describe('Notification History Component', () => {
     let notificationService: NotificationService;
     let overlayContainerElement: HTMLElement;
     let storage: StorageService;
+    let testNotifications: NotificationModel[];
 
     const openNotification = () => {
         fixture.detectChanges();
         const button = element.querySelector<HTMLButtonElement>('#adf-notification-history-open-button');
-        button.click();
+        button?.click();
         fixture.detectChanges();
     };
 
+    const getMatBadgeElement = (): HTMLElement => fixture.debugElement.query(By.css('[matBadge]')).nativeElement;
```

Comment:
we should not be addressing the material classes directly, already mentioned million times

Reply:
removed as it is not needed anymore. Related test case is removed